lib/Fhp/Model/TanRequestChallengeFlicker/FlickerTanStartCode.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace Fhp\Model\TanRequestChallengeFlicker;
+
+use InvalidArgumentException;
+
+class FlickerTanStartCode extends FlickerTanDataElement
```

Comment:
Here and elsewhere, please link to https://www.hbci-zka.de/dokumente/spezifikation_deutsch/hhd/Belegungsrichtlinien%20TANve1.5%20FV%20vom%202018-04-16.pdf
The other specifications don't contain important details like that the length fields should be decimal.